### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new article listing all the products we integrate with:
https://help.sumologic.com//docs/integrations/product-list

Issue number: Asana ticket "[Create article showing all the products we integrate with](https://app.asana.com/0/1183741177809740/1205598772384150/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft.

**Comment by jpipkin1** [comment]
This PR is ready for review.

File: docs/integrations/product-list.md [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+|  <img src={useBaseUrl('img/integrations/misc/doppler-logo.png')} alt="Thumbnail icon" width="50"/>   | [Doppler](https://www.doppler.com/)  | Partner integration:	[Doppler](https://docs.doppler.com/docs/sumologic) | 
+|  <img src={useBaseUrl('img/integrations/saas-cloud/dropbox-icon.svg')} alt="Thumbnail icon" width="50"/>   |  [Dropbox](https://www.dropbox.com/) | App: [Dropbox](/docs/integrations/saas-cloud/dropbox/)	<br/>Collector: [Dropbox Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source/) | 
+|  <img src={useBaseUrl('img/send-data/druva-logo.png')} alt="Thumbnail icon" width="75"/>   | Druva  | 	Apps:<br/>- [Druva](/docs/integrations/saas-cloud/druva/)	<br/>- [Druva Cyber Resilience](/docs/integrations/saas-cloud/druva-cyber-resilience/)	<br/>Collectors: <br/>- [Druva Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source/) <br/>- [Druva Cyber Resilience Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source/) |
```
Comment (kimsauce) [inline docs/integrations/product-list.md:223]: cleaner image on file

```suggestion
|  <img src={useBaseUrl('img/send-data/druva-logo.svg')} alt="Thumbnail icon" width="75"/>   | Druva  | 	Apps:<br/>- [Druva](/docs/integrations/saas-cloud/druva/)	<br/>- [Druva Cyber Resilience](/docs/integrations/saas-cloud/druva-cyber-resilience/)	<br/>Collectors: <br/>- [Druva Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source/) <br/>- [Druva Cyber Resilience Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source/) | 
```

Comment (jpipkin1) [reply docs/integrations/product-list.md]: Done! See commit "[Remove druva-logo.png file](https://github.com/SumoLogic/sumologic-documentation/pull/3222/commits/41f796c8f085e2bb6169e696b8afd0dd18e9f418)".

**Review by kimsauce — APPROVED** [review]
Freakin' perfect!!

**Comment by jpipkin1** [comment]
Gee whiz, @kimsauce! Thanks!

**Pull request merged**